build.gradle

```diff
@@ -102,6 +102,12 @@ allprojects {
             force "com.google.guava:guava:" + guavaVersion
             force "org.glassfish.jersey.core:jersey-common:" + jerseyVersion
             force "org.glassfish.jersey.core:jersey-server:" + jerseyVersion
+            force "org.apache.zookeeper:zookeeper:" + apacheZookeeperVersion
+            force "io.netty:netty-transport:"+ nettyVersion
+            force "io.netty:netty-handler:" + nettyVersion
+            force "io.netty:netty-transport-native-epoll:" + nettyVersion
+            force "io.netty:netty:" + nettyAllVersion
```

Comment:
https://mvnrepository.com/artifact/io.netty/netty  this artifact has been moved to `netty-all`

Reply:
The fact is that it gets (transitively?) downloaded in master and in the branch of this PR the version changes. I forced its version, which can be seen a too defensive strategy. Maybe it would be better to remove that dependency if possible.